seqr-gke-nodepool/main.tf

```diff
@@ -31,6 +33,17 @@ resource "google_container_node_pool" "node_pool" {
       enable_integrity_monitoring = "true"
       enable_secure_boot          = "false"
     }
+
+    workload_metadata_config {
```

Comment:
the `default` node pool is configured in the `seqr-gke` module and already has this flag.